### PR TITLE
S390x 2.3 trivial fixes [v2]

### PR DESCRIPTION
**Pull request by clebergnu**
This is a collection of fixes and small improvements found while reviewing @ldoktor PR #1170 .

Changes from v1 (#1173):
 - Based, and rebased, on v3
 - Remove comments from iscsi target configuration
 - Added missing `astring` import

### CODE REVIEW

File: qemu/tests/cfg/boot_from_device.cfg [hunk 1]
```diff
@@ -76,9 +76,10 @@
         - boot_from_iscsi_device:
             start_vm = no
             dev_name = iscsi-dev
-            portal_ip = 10.66.90.100
-            initiator = "iqn.2010-07.com.redhat:kvmautotest"
-            target = "iqn.2001-05.com.equallogic:0-8a0906-db31f7d03-470263b05654c204-kvm-puyiqiao"
+            # Please update these to point to your iscsi server
+            portal_ip = 127.0.0.1
+            initiator = "iqn.2010-07.com.example:my_initiator"
+            target = "iqn.2001-05.com.mytarget:0-0a0000-000000000-0000000000000000-my-target"
```
Comment (ldoktor) [inline qemu/tests/cfg/boot_from_device.cfg]: Does including those first 4 patches in your PR means they are ACKed by you? If so we can merge https://github.com/autotest/tp-qemu/pull/1175 as the shas are the same and @luckyh already gave it first ACK.

File: qemu/tests/multi_disk.py [hunk 2]
```diff
@@ -209,7 +210,7 @@ def _get_disk_index(session, image_size, disk_indexs):
     if params.get("multi_disk_params_only") == 'yes':
         # Only print the test param_matrix and finish
         logging.info('Newly added disks:\n%s',
-                     utils.matrix_to_string(param_table, param_table_header))
+                     astring.tabular_output(param_table, param_table_header))
```
Comment (ldoktor) [inline qemu/tests/multi_disk.py]: ACK

File: qemu/tests/multi_disk.py [hunk 3]
```diff
@@ -368,7 +369,7 @@ def _get_disk_index(session, image_size, disk_indexs):
                 disks = re.findall(re_str, output)
                 disks.sort()
                 for disk in disks:
-                    error.context("Unmounting disk: %s..." % disk)
+                    error_context.context("Unmounting disk: %s..." % disk)
```
Comment (ldoktor) [inline qemu/tests/multi_disk.py]: ACK

File: qemu/tests/cfg/qemu_img_negative.cfg [hunk 4]
```diff
@@ -18,14 +18,14 @@
...
                     images += " sn1 sn2"
                     image_name_image1 = "lv_base"
                     image_name_sn1 = "lv_sn1"
                     image_name_sn2 = "lv_sn2"
                     rebase_list = "sn2 > image1"
                     negtive_test_sn2 = yes
+                    #Please update portal_ip, initiator and target params based on your test env
+                    portal_ip = 127.0.0.1
+                    initiator = "iqn.2010-07.com.example:my_initiator"
+                    target = "iqn.2001-05.com.mytarget:0-0a0000-000000000-0000000000000000-my-target"
```
Comment (ldoktor) [inline qemu/tests/cfg/qemu_img_negative.cfg]: ack

File: qemu/tests/live_snapshot_chain.py [hunk 5]
```diff
@@ -79,97 +98,99 @@ def do_operate(params, key_word):
...
+        errs = cleanup_images(snapshot_chain, params)
+        test.assertFalse(errs, "Errors occurred while removing images:\n%s"
+                         % "\n".join(errs))
+    except Exception as details:
+        error.context("Force-cleaning after exception: %s" % details,
+                      logging.error)
+        if vm.is_alive():
+            vm.destroy()
+        cleanup_images(snapshot_chain, params)
+        raise
```
Comment (apahim) [inline qemu/tests/live_snapshot_chain.py]: ok, looks fine.

File: qemu/tests/cgroup.py [hunk 6]
```diff
@@ -182,16 +182,18 @@ def param_add_scsi_disks():
         Adds scsi_debug disk to every VM in params['vms']
         :param prefix: adds prefix to drive name
         """
-        if process.system("lsmod | grep scsi_debug", ignore_status=True):
+        if process.system("lsmod | grep scsi_debug", ignore_status=True,
```
Comment (apahim) [inline qemu/tests/cgroup.py]: ok

File: qemu/tests/cgroup.py [hunk 7]
```diff
@@ -604,8 +604,8 @@ def _test(direction, blkio):
         # ; true is necessarily when there is no dd present at the time
         kill_cmd = "rm -f /tmp/cgroup_lock; killall -9 dd; true"
         stat_cmd = "killall -SIGUSR1 dd; true"
-        re_dd = (r'(\d+) bytes \(\d+\.*\d* \w*\) copied, (\d+\.*\d*) s, '
-                 '\d+\.*\d* \w./s')
+        re_dd = (r'(\d+) bytes \(\d+\.*\d* \w*(, \d+\.*\d* \w*)?\) copied, '
+                 '(\d+\.*\d*) s, \d+\.*\d* \w./s')
```
Comment (apahim) [inline qemu/tests/cgroup.py]: yes, good one.

File: qemu/tests/cfg/boot_from_device.cfg (hunk 1 above)
Comment (apahim) [inline qemu/tests/cfg/boot_from_device.cfg]: ok

**Comment by ldoktor** [comment]
OK I see 2 acks so let's get this in...

**Pull request merged**